src/System.Data.Common/src/System/Data/Common/DbDataReader.cs

```diff
@@ -23,6 +23,11 @@ protected DbDataReader() : base()
         {
         }
 
+        ~DbDataReader()
```

Comment:
This is a behavioral change from the full framework.  I'm surprised to see this in the same commit / PR as the others, as it's not clear to me this is actually a desired change.